### PR TITLE
Add feature to deny all feedback for a given user

### PR DESCRIPTION
**Pull request by chrishtr**
The WBE site currently has a lot of spam from a user named "pHqghUme". Either someone attacking our server or a script gone rogue...

So I added this feature to clean it up with one click.

### CODE REVIEW

File: infra/api.py [hunk 1]
```diff
@@ -100,6 +100,10 @@ def contributors(self):
         }
 
     def set_status(self, i, status):
+        if status == "denied-all":
+           for d in self.data:
+                if d['name'] == self.data[i]['name']:
+                    d['status'] = 'denied'
         self.data[i]['status'] = status
```
Comment (pavpanchekha) [inline infra/api.py:107]: Doesn't this then set the one you denied-all to have a weird new status?

File: www/entries.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ function feedback_mode() {
         elts[i].addEventListener('click', function() {
             var pw = window.localStorage["pw"];
             submit_status(parseInt(this.parentNode.dataset.id), this.className, pw);
-            this.parentNode.remove();
+            location.reload();
```
Comment (pavpanchekha) [inline www/entries.js:13]: I assume this is fast enough to not be annoying?

**Review by pavpanchekha — APPROVED** [review]

File: www/entries.js (hunk 2 above)
Comment (chrishtr) [reply www/entries.js:13]: > I assume this is fast enough to not be annoying?

I think it should be. Currently there are a ton of entries on the feedback page and it loads pretty fast.

File: infra/api.py (hunk 1 above)
Comment (chrishtr) [reply infra/api.py:107]: Oops, forgot a return statement. Added

**Pull request merged**